New package: OceanLight v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: OceanLight
- Repository: https://github.com/haoboatlab/OceanLight.jl
- Created by: @tayT0T
- Version: v1.0.0
- Commit: 4e99228d33b03d6643795c1f753652eebb56002d
- Reviewed by: @tayT0T
- Reference: https://github.com/haoboatlab/OceanLight.jl/issues/3#issuecomment-2423064800
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598fd82be2f3b82ffa160a1c69208bb315704df2101c7f083df407de2709c79a2ab5713ef0458533b552ac7c1a2b21e7621431126809cc20760aa3476954cad02bb3d68c1e8eabdcaf7d432dcf4a3bd9e8d6321188dd47bfeda465959f55850c47445b13a552f8eb9028a4756ec5e3ddeefcbeeb07dc1ffc8e887fc15c22b955731 -->

goerz commented:
[noblock] Documenter should probably not be a dependency of your project. Please make sure to only list the minimum number of dependencies

JuliaRegistrator committed:
New package: OceanLight v1.0.0

UUID: ff88e13d-0366-4369-baeb-7c3fb86ee06b
Repo: https://github.com/haoboatlab/OceanLight.jl.git
Tree: a343b821768d3846e160e77e49f492a397ad1cbf

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
